docs/index.md

```diff
@@ -62,7 +62,7 @@ ROCm documentation is organized into the following categories:
 * [File structure (Linux FHS)](./conceptual/file-reorg.md)
 * [GPU isolation techniques](./conceptual/gpu-isolation.md)
 * [Using CMake](./conceptual/cmake-packages.rst)
-* [ROCm & PCIe atomics](./conceptual/More-about-how-ROCm-uses-PCIe-Atomics.rst)
+* [ROCm & PCIe atomics](./conceptual/pcie-atomics.rst)
```

Comment:
Another nitpick regarding the TOC: 
The Google developer documentation style guide says to avoid `&`. https://developers.google.com/style/words-as-words

Not a dealbreaker though

Reply:
Updated the filename